addons/binding/org.openhab.binding.chromecast/README.md

```diff
@@ -34,7 +34,13 @@ No authentication is required for accessing the devices on the network.
 ## Thing Configuration
 
 Chromecast devices can also be manually added.
-The only configuration parameter is the `ipAddress`.
+The only configuration parameter is the `ipAddress`. For an audio group also the port is necessary. The autodiscover process finds the port automatically. With manual thing configuration the port must be determined manually.
```

Comment:
One typo: "autodiscovery"

```suggestion
The only configuration parameter is the `ipAddress`.
For an audio group also the port is necessary.
The autodiscovery process finds the port automatically.
With manual thing configuration the port must be determined manually.
```

Reply:
"also the **por" :)

Reply:
I edited my comment. Thanks.

Reply:
@miwok72 Did you see my comment? Do you want to apply the suggestion or not?

Reply:
Thank you, please apply your suggestion!

Reply:
@miwok72 May I ask you to do it?

Reply:
I'm sorry, that was my first PR and I'm not sure which steps to take

Reply:
No problem. In https://github.com/openhab/openhab2-addons/pull/4946#discussion_r258961705 you should see a "Commit suggestion" button. Just click on it and the suggestion will be applied.